adhocracy4/maps/mixins.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from easy_thumbnails.files import get_thumbnailer
+
+
+class MapItemListMixin(object):
+
+    def dump_geojson(self):
+        result = {}
+        result['type'] = 'FeatureCollection'
+        feature_list = []
+
+        for item in super().get_queryset():
+
+            url = ''
+
+            if hasattr(item, 'image') and item.image:
+                image = get_thumbnailer(item.image)['map_thumbnail']
+                url = image.url
+
+            properties = {
+                'name': item.name,
```

Comment:
At on `name` here (and maybe also with `slug`, `image` and `url`) there should be a call to escape.